# BaseVectorGeod block 
# Block_Number 4028

std_msgs/Header header

# SBF block header including time header
BlockHeader block_header

uint8   n
uint8   sb_length

VectorInfoGeod[] vector_info_geod 